### DISCUSSION TITLE
Frsky Archer receivers will not register

### DISCUSSION THREAD

**Discussion by Richjanz** [EdgeTX/edgetx#3151] [General]
I have recently flashed my Horus x10s express from OpenTx to EdgeTx 2.8. I also had the models converted to Edge. However now I have lost my receivers. I have 3 Frysky Archer 8R Pros and one 10R Pro. None of them will register so I cannot bind them. I tried all sorts of ways to register them to no avail. I'm wondering if I should go back to OpenTx  2.3.15 and then start again with Edge

**Comment by pfeerick** [comment]
Funnily enough, I actually had an Archer GR8 out earlier today to test something, and was able to both register and bind it on X9D+2019 running 2.8, so it may be something specific to the colorlcd side of things.

**Reply by rolfz** [reply]
me too please fix it it worked under opentx i have rx6r

**Reply by rolfz** [reply]
what i need to do to my hw

**Comment by Richjanz** [comment]
If I go to the airplanes that were working and turn them on the receiver LED is red asif it waiting for the tx. I wonder if there is a problem with the internal RF ,ISRM module

**Comment by Richjanz** [comment]
Problem solved. I replaced the internal ISRM R/F card and everything is working as it should. I thank everyone for their input with this problem. Turned out to be a hardware problem.

**Reply by pfeerick** [reply]
Damn. Sorry to hear that. Glad you got it working though! :)

**Reply by rolfz** [reply]
i do not understand how to fix the problem  have you a picture of the hw fix is the fix on the tx side of my qx7

**Comment by rolfz** [comment]
Great what do i need to load on my taranisEnvoyé de mon iPhoneLe 27 déc. 2023 à 17:19, Richjanz ***@***.***> a écrit :﻿
Closed #3151 as resolved.

**Comment by rolfz** [comment]
how can i make it work for me ??

**Comment by pfeerick** [comment]
You can start off by explaining what hardware you are using... what
transmitter? You mention taranis and rx6r, so an ACCESS
compatible receiver... I'm *guessing* Taranis X9D+2019? What version of
EdgeTX are you using? Are the receivers indicating they don't have a link
with the transmitter at all? Is it registration or binding that is the
problem?

The person who had issues before had a Horus X10S Express, and
unfortunately it seems like the internal RF failed, so the ISRM RF board
needed replacing.

**Comment by rolfz** [comment]
Thank you for the Answer, I have Taranis QX7 tx and the latest edgetx 929 fwLets hope this helpsRolfEnvoyé de mon iPhoneLe 28 déc. 2023 à 10:34, Peter Feerick ***@***.***> a écrit :﻿
You can start off by explaining what hardware you are using... what

transmitter? You mention taranis and rx6r, so an ACCESS

compatible receiver... I'm *guessing* Taranis X9D+2019? What version of

EdgeTX are you using? Are the receivers indicating they don't have a link

with the transmitter at all? Is it registration or binding that is the

problem?



The person who had issues before had a Horus X10S Express, and

unfortunately it seems like the internal RF failed, so the ISRM RF board

needed replacing.

**Comment by pfeerick** [comment]
When you say Q X7, this is the ACCESS version (not the ACCST version)? And
when you flashed EdgeTX v2.9.2, you used the Frsky QX7 ACCESS option? When
you try to enable the internal RF, does it give you ISRM or XJT as the
option?
